DANS-KNAW/easy-stage-dataset
# EASY-1055: Get depositorId from deposit.properties rather than from c…

fixes EASY-1055
#### When applied it will

Get depositorId from deposit.properties rather than from configuration file.
#### Where should the reviewer @DANS-KNAW/easy start?

Setting.scala and then Conf.scala 
#### How should this be manually tested?
- mvn clean install of the project
- feed the created target/test/sdoPuddings one by one to easy-ingest-flow or easy-ingest
- use the webui to verify the created datasets (ids logged by the previous step)
#### related pull requests on github

| repo | PR |
| --- | --- |
| easy-ingest-flow | [42#](https://github.com/DANS-KNAW/easy-ingest-flow/pull/41) |
